lib/workers/repository/master-issue.js

```diff
@@ -138,5 +138,14 @@ async function ensureMasterIssue(config, branches) {
     }
     issueBody += '\n';
   }
+  const alreadyOpened = branches.filter(branch => branch.res === 'done');
```

Comment:
Better to reuse this variable instead: https://github.com/renovatebot/renovate/blob/5e59756cf218d62fa7e0aa3a321dae92bf35d4e0/lib/workers/repository/master-issue.js#L113

Reply:
Test if inProgress.length > 1

Reply:
inProgress contain more than just the opened branches, it also contains the branches which is not created yet

Reply:
This is the array we use to populate the “Open” list right now so it seems like the best one to use here too

Reply:
ok, I will use this